src/graphics.rs

```diff
@@ -8,6 +8,19 @@ use crate::models::Model;
 use crate::{Display, Error, Orientation};
 use display_interface::WriteOnlyDataCommand;
 
+fn framebuffer_size<DI, RST, M>(display: &Display<DI, RST, M>) -> (u16, u16)
```

Comment:
Thought about making this return a `Size` object. That would clarify which one is width vs height. But Size objects use u32 dimensions. And `pixel_count` below uses `usize` dimensions. And `usize::from` works fine for u16, but fails for u32. So I stuck with the above for now.